examples/hybrid/sphere/remap_pipeline.jl

```diff
@@ -17,6 +17,12 @@ else
     error("ENV[\"THERMO_VAR\"] require (\"e_tot\" or \"theta\")")
 end
 
+if haskey(ENV, "MOIST_MODE")
+    moist_mode = ENV["MOIST_MODE"]
+else
+    error("ENV[\"MOIST_MODE\"] require (\"dry\" or \"equil\")")
+end
```

Comment:
Could we switch these to kwargs in `remap2latlon` and call this in our CI?

Reply:
(not a blocker, it'd just be nice to see this in action on CI!)

Reply:
> Could we switch these to kwargs in `remap2latlon` and call this in our CI?

Yes, that will be great! I am actually thinking of a way to remap in the callback setup so that we could have a proper nc outputs while running the simulation.